cmd-common/src/main/kotlin/com/gojuno/cmd/common/Processes.kt

```diff
@@ -11,9 +11,9 @@ import java.util.concurrent.TimeUnit
 import java.util.concurrent.TimeUnit.SECONDS
 import java.util.concurrent.TimeoutException
 
-val home: String = System.getenv("HOME")
-val androidHome: String = System.getenv("ANDROID_HOME")
-val adb: String = "$androidHome/platform-tools/adb"
+val home: String by lazy { System.getenv("HOME") }
+val androidHome: String by lazy { System.getenv("ANDROID_HOME") }
```

Comment:
Was failing build if `ANDROID_HOME` is not set even if we don't touch `androidHome` in tests.